Simplenote/TagListViewController.swift

```diff
@@ -576,15 +596,19 @@ private extension TagListViewController {
     }
 
     func refreshTableHeaderView() {
-        guard let headerView = tableView.tableHeaderView as? SustainerView else {
+        guard let sustainerHeaderView else {
+            return
+        }
+
+        if isActiveSustainer {
+            tableView.tableHeaderView = nil
```

Comment:
We're, now, hiding the Header in the Tags List, whenever there's an active subscription